site/resources/js/enhancement.js

```diff
@@ -594,6 +594,31 @@ function toggleInputGroupFocus(event) {
 	}
 }
 
+
+function deleteTag(button) {
```

Comment:
@Rajdeepc This is great but last we spoke about this, I suggested holding off until we finish our 11ty migration because these files are all being rewritten in the process.